.travis.yml

```diff
@@ -1,14 +1,16 @@
-sudo: required
-
-# any language should do it
 language: php
 
-matrix:
+# Testing Ubuntu 18.04
+os: linux
+dist: bionic
+
+jobs:
   include:
   - php: "7.3"
   - php: "7.4"
 
 services:
+- mysql
```

Comment:
This is not necessary. Docker will create mysql service for testing.

Reply:
It has changed for 18.04. It explains in the link

Reply:
"On the Ubuntu 18.04 based environment, to speed up boot time and improve performance we’ve disabled all services by default. Add any services that you want to start to your .travis.yml:  "
Integration testing only needs mysql service from docker system not from travis.

Reply:
The link just says they've disabled all services by default (that I see).. I think @kongtiaowang's point is that we don't use the Travis mysql service, it gets launched from docker.